lib/src/server_messages.dart

```diff
@@ -133,6 +133,19 @@ class DataRowMessage extends ServerMessage {
   String toString() => "Data Row Message: ${values}";
 }
 
+class NotificationResponseMessage extends ServerMessage {
+  int processId;
+  String channel;
+  String payload;
+
+  void readBytes(Uint8List bytes) {
+    var view = new ByteData.view(bytes.buffer, bytes.offsetInBytes);
+    processId = view.getUint32(0);
+    channel = UTF8.decode(bytes.sublist(4, bytes.indexOf(0, 4)));
```

Comment:
You can avoid a copy (`bytes.sublist`) on this line and the next by using `UTF8.decoder.startChunkedConversion` and piping it bytes from the `view`.